src/objects/Hierarchy_Settings__c.object

```diff
@@ -390,6 +390,16 @@
         <type>Text</type>
         <unique>false</unique>
     </fields>
+    <fields>
+        <fullName>Validate_Program_Plan_for_Nested_PR__c</fullName>
+        <defaultValue>true</defaultValue>
+        <description>When selected, Plan Requirements can specify either a Program Plan or a parent Plan Requirement, but not both. For example, a Plan Requirement that has a parent Plan Requirement can’t also specify a Program Plan, or vice versa. If you specify both a Program Plan and a parent Plan Requirement, validation fails and the Plan Requirement isn’t saved.</description>
+        <externalId>false</externalId>
+        <inlineHelpText>When selected, Plan Requirements can specify either a Program Plan or a parent Plan Requirement, but not both. For example, a Plan Requirement that has a parent Plan Requirement can’t also specify a Program Plan, or vice versa. If you specify both a Progr</inlineHelpText>
```

Comment:
@xinnanlisf the inline help here is getting cut off. @juliesnow do we want to do anything about this?

Reply:
Yeah, @lifewithryan - Xinnan will remove "For example, a Plan Requirement that has a parent Plan Requirement can’t also specify a Program Plan, or vice versa," which pulls it well below the 255 character limit.